src/app/pages/TopicPage/Curation/Subhead/index.styled.tsx

```diff
@@ -0,0 +1,41 @@
+import styled from '@emotion/styled';
+
+import { getDoublePica } from '../../../../legacy/psammead/gel-foundations/src/typography';
+
+import { getSansBold } from '../../../../legacy/psammead/psammead-styles/src/font-styles';
+import {
+  C_POSTBOX,
+  C_GREY_10,
+} from '../../../../legacy/psammead/psammead-styles/src/colours';
+
+export default styled.a<{ service: string; script: string }>`
```

Comment:
Not sure of our standards (if any) on how we declare types/interfaces, but I would maybe extract this out to make it a bit more readable.

Reply:
👍 yeah, was going to comment on the coding standards that there's no typescript stuff there - should probably agree and document those standards (probably as a separate bit of work, to give Jonathan a rest!)

Reply:
I left a comment about if we should have separate type declaration files and whether to use types or interfaces so would be good to hash that out in more detail within the standards

Reply:
Refactored! put the types into a separate Props interface for now